test/mpi.py

```diff
@@ -195,7 +195,7 @@ def test_select_device(self):
     def test_make_node_file(self):
         """test if the nodefile is made and if it contains the same amount of nodes as mpinodes"""
         mpi_instance = getinstance(mpim.MPI, Local, MympirunOption())
-        mpi_instance.make_node_file()
+        mpi_instance.make_machine_file()
```

Comment:
@boegel to be fair, we never did test the mpdbootfile part of make_node_file. i just split make_node_file into machine file and mpdbootfile seperately